src/Compute/Compute.Test/ScenarioTests/StrategiesVirtualMachineTests.ps1

```diff
@@ -452,11 +452,13 @@ function Test-SimpleNewVmImageName
         [string]$domainNameLabel = "$vmname-$vmname".tolower()
 
         # Common
-        $x = New-AzVM `
+        $imgversion = Get-VMImageVersion -publisher "MicrosoftWindowsServer" `
+							-offer "WindowsServer" -sku "2016-Datacenter"
+		$x = New-AzVM `
             -Name $vmname `
             -Credential $cred `
             -DomainNameLabel $domainNameLabel `
-            -ImageName "MicrosoftWindowsServer:WindowsServer:2016-Datacenter:2016.127.20170406"
+            -ImageName ("MicrosoftWindowsServer:WindowsServer:2016-Datacenter:" + $imgversion)
```

Comment:
Can't we just use 'latest' here as the image version?

Reply:
Yes. Another test is using latest